lms/templates/dashboard/_dashboard_course_listing.html

```diff
@@ -202,6 +202,14 @@ <h3 class="course-title" id="course-title-${enrollment.course_id}">
                   </a>
                 % endif
               % endif
+            % elif hasattr(show_courseware_link, 'user_message'):
+              <span class="enter-course-blocked"
+                 data-course-key="${enrollment.course_id}">
+                ${show_courseware_link.user_message}
```

Comment:
How do we handle translation/localization of the error message?

Reply:
The message template strings should be translated (none of those in this PR, but there will be some coming in future PRs)